design-system/src/theme-provider.visualroute.jsx

```diff
@@ -16,7 +16,11 @@ export const routePath = '/theme-provider';
 const parentSelector = (id) => () => document.getElementById(id);
 
 const DummyComponent = (props) => {
-  const theme = useTheme(parentSelector(props.parentId));
+  const { currentTheme } = useTheme(
+    props.parentId
+      ? parentSelector(props.parentId)
+      : () => document.documentElement
```

Comment:
Do we need this given that `useTheme` has a `defaultParentSelector` function?

```suggestion
      : undefined
```

Reply:
Updated in this commit: bc1aca29d24e65d565b04c30f86c69a4872c8ef8